### ISSUE TITLE
discord changed the profile cards

### ISSUE THREAD

**Issue by NeonVolta** [Goose-Nest/GT-RevertRebrand#29]
they need to be reverted

**Comment by Fuzuki785** [comment]
This change is unrelated to the rebranding.

**Comment by SupDos** [comment]
Seems like it was reverted anyways (https://github.com/Goose-Nest/GT-RevertRebrand/commit/0fc361c71f21849eeffd76c1e6d461fd8673609c)

Not sure why as Hades said, this is unrelated to the rebranding. @CanadaHonk could you clarify?

**Comment by mutageneral** [comment]
> This change is unrelated to the rebranding.

The rebranding is basically every change's discord do since they announced the rebranding. Discord are going to change pretty much everything until it becomes a completely new look, the rebranded look. So yes, the new profile cards would need to be reverted.

> Seems like it was reverted anyways

The revert is closer to how it used to look but it is clearly not how it used to look. I note the colour is wrong, and the new banner is visible too. I think there might also be shape and spacing differences.

Revert:
![image](https://user-images.githubusercontent.com/62779748/119921895-cd67b180-bf3c-11eb-8dff-ba539a01faea.png)
Original (not my image, found it with an image search):
![image](https://user-images.githubusercontent.com/62779748/119922845-a5794d80-bf3e-11eb-9cc2-99649e8dc07e.png)

**Comment by Fuzuki785** [comment]
> The rebranding is basically every change's discord do since they announced the rebranding.

The rebranding is changes related to Discord's brand, so font, logo, colours, and so on. Not features added later. Otherwise everything from now on will just be "the rebranding".

Profile customisation is an entirety separate feature, which is not defined by Discord's brand guidelines or anywhere else. It's like saying that server screen sharing should be reverted if it happened after the rebranding. I understand that you may not like the new profile pop-outs, but they're **not** part of the rebranding process, and they are not even finalised. It's still a beta feature, that's starting to be deployed, so they may still change (although they probably won't).

As it's not a part of the rebranding, it doesn't really fit into a theme made to revert it, in my opinion. But people are still allowed to write themes that will change other parts of Discord, if you want to remove everything from Discord until there's just the chat history, who am I to stop you? But that wouldn't fit in a theme like this one, which doesn't have this purpose.

**Comment by mutageneral** [comment]
> > The rebranding is basically every change's discord do since they announced the rebranding.
> 
> The rebranding is changes related to Discord's brand, so font, logo, colours, and so on. Not features added later. Otherwise everything from now on will just be "the rebranding".
> 
> Profile customisation is an entirety separate feature, which is not defined by Discord's brand guidelines or anywhere else. It's like saying that server screen sharing should be reverted if it happened after the rebranding. I understand that you may not like the new profile pop-outs, but they're **not** part of the rebranding process, and they are not even finalised. It's still a beta feature, that's starting to be deployed, so they may still change (although they probably won't).
> 
> As it's not a part of the rebranding, it doesn't really fit into a theme made to revert it, in my opinion. But people are still allowed to write themes that will change other parts of Discord, if you want to remove everything from Discord until there's just the chat history, who am I to stop you? But that wouldn't fit in a theme like this one, which doesn't have this purpose.

I disagree. Maybe the banner could be kept because it is a "feature" but the change of the back colour of it is clearly inline with their rebranding. You cannot compare the colour change of the user modals with the feature of screen sharing. If you personally want to turn off the changes done to the user modals you can follow the tutorial I posted in issue #24. The only difference to the tutorial is that you need to find "Revert new user modals" instead of "Revert reactions".

**Comment by Fuzuki785** [comment]
> the change of the back colour of it is clearly inline with their rebranding

I agree with this.

> You cannot compare the colour change of the user modals with the feature of screen sharing.

I wasn't, I was comparing feature to feature, since it was my understanding that people wanted the new profile customisation feature to be gone with the theme.

> If you personally want to turn off the changes done to the user modals you can follow the tutorial I posted in issue #24.

I don't use RevertRebrand myself, but it seemed important for me to make the distinction between what is and isn't part of the rebranding, for the future, since people have been asking for this same thing on the  GooseNest Discord server for a few days.

**Comment by CanadaHonk** [comment]
Already partially done but will be finished off soon, will be made disabled by default / optional.

**Comment by KingGamingYT** [comment]
I'm not sure If a Discord update broke it but the toggle no longer seems to do anything.

**Comment by milbits** [comment]
put this in custom css:
```css
.body-3HBlXn {
  background: #2F3136;
}
.bodyInnerWrapper-26fQXj, .footer-3UKYOU {
  background: #2F3136;
}
.headerTop-3vNv-a  {
  background: #202225;
}
.headerTop-3vNv-a headerTopWithCustomStatus-1DHAjX {
  background: #202225;
}
.textBadge-1iylP6 {
  display: none;
}
.avatar-37jOim {
  background: #202225;
}
```
this only reverts the colors but better than nothing

**Comment by KingGamingYT** [comment]
![image](https://user-images.githubusercontent.com/45918062/130719779-8c855c2c-d142-4555-9e34-a8c42ec4d082.png)

I agree, it is indeed better than nothing. Thanks!

**Comment by Compwnter** [comment]
> 
> 
> put this in custom css:
> 
> ```css
> .body-3HBlXn {
>   background: #2F3136;
> }
> .bodyInnerWrapper-26fQXj, .footer-3UKYOU {
>   background: #2F3136;
> }
> .headerTop-3vNv-a  {
>   background: #202225;
> }
> .headerTop-3vNv-a headerTopWithCustomStatus-1DHAjX {
>   background: #202225;
> }
> .textBadge-1iylP6 {
>   display: none;
> }
> .avatar-37jOim {
>   background: #202225;
> }
> ```
> 
> this only reverts the colors but better than nothing

I wouldn't add this to RevertRebrand but it's very nice either way.

**Comment by Compwnter** [comment]
> 
> 
> > put this in custom css:
> > ```css
> > .body-3HBlXn {
> >   background: #2F3136;
> > }
> > .bodyInnerWrapper-26fQXj, .footer-3UKYOU {
> >   background: #2F3136;
> > }
> > .headerTop-3vNv-a  {
> >   background: #202225;
> > }
> > .headerTop-3vNv-a headerTopWithCustomStatus-1DHAjX {
> >   background: #202225;
> > }
> > .textBadge-1iylP6 {
> >   display: none;
> > }
> > .avatar-37jOim {
> >   background: #202225;
> > }
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > this only reverts the colors but better than nothing
> 
> I wouldn't add this to RevertRebrand but it's very nice either way.

+ I wouldn't add it due to it not being part of the "Imagine a place" redesign.
